docs/strategies/hedging.md

```diff
@@ -323,7 +323,7 @@ new ResiliencePipelineBuilder<HttpResponseMessage>()
             {
                 0 => TimeSpan.FromSeconds(1),
                 1 => TimeSpan.FromSeconds(2),
-                _ => System.Threading.Timeout.InfiniteTimeSpan
+                _ => TimeSpan.FromSeconds(-1)
```

Comment:
I feel we need a comment somewhere explaining what -1 signifies.

Reply:
At the [fallback mode](https://www.pollydocs.org/strategies/hedging.html#fallback-mode) section it is explicitly stated:

> In fallback mode, the Delay value should be less than TimeSpan.Zero.

The reason of the replacement: The `InfiniteTimeSpan` is a bit misleading because it represent -1 milliseconds. It might not be obvious why `InfiniteTimeSpan < TimeSpan.Zero`

Reply:
I feel that we should still add a small comment to the code snippet reiterating that.

On my laptop it's about four full pages of scrolling down between that sentence and the code snippet, and it's mentioned in a completely different section to where it's used (Fallback mode vs Dynamic mode).

I think it would be easy for someone to skim read (or go straight to the Dynamic section) and then not understand the significance.

Reply:
@martintmk This ` 0 => TimeSpan.FromSeconds(1)` line should be ` 0 => TimeSpan.Zero` to align with description below the code, right?

Reply:
Correct, the sample should be:

``` csharp
0 => TimeSpan.Zero,
1 => TimeSpan.Zero,
_ => TimeSpan.FromSeconds(-1)
```

Reply:
Let me fix in this PR, then

Reply:
Could change to `0 or 1 => TimeSpan.Zero,`?

Reply:
Yepp 👍